torchci/lib/bot/autoLabelBot.ts

```diff
@@ -216,55 +174,65 @@ function myBot(app: Probot): void {
       topic = "topic: deprecation";
     }
 
+    // these files do not warrant a real category and mostly not user facing
+    // we want to return this _before_ categorizing
+    if (isNotUserFacing(filesChanged) || title.toLowerCase().includes("[codemod]")) {
+      topic = "topic: not user facing";
+    }
+
+    // don't re-categorize those with existing labels: already topiced
+    if (labels.some(l => l === "topic: not user facing")) {
+        return  "skip";
+    }
+    return topic;
+  }
+
+  // https://github.com/pytorch/pytorch/blob/master/scripts/release_notes/commitlist.py#L90
+  async function getReleaseNotesCategory(
+    title: string,
+    labels: string[],
+    filesChanged: string[],
+  ): Promise<string> {
```

Comment:
These config settings are all the tricky ones to move to a yaml file.

It would be great to come up with a way to stick these into the yaml file too (and we can iterate on what a good format might look like).

cc: @malfet